resources/lang/en/microtasking.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+return [
+    'title' => 'Upstream',
+
+    'news' => [
+        'title' => 'Read about our recent success',
+        'content' => 'Through our FaultCat quest, hundreds of volunteers helped us analyse our laptop data.
+                    <ul>
+                        <li><a href="">Read the results</a> as they went into this policy paper on laptop lifetimes.</li>
+                        <li>
+                            <a href="">Check out the data visualisation</a> featured in The Times newspaper.
+                        </li>
+                    </ul>
+        ',
+    ],
+    'volunteering' => [
+        'title' => 'Volunteer from anywhere',
+        'content' => '            <p>
+                We collect data at community events.  But the raw data needs your help to become strong evidence for better products.  With your help, we can change the system.
+            </p>
+            <p>
+               This is an easy way to make a difference, whether or not you can join us at our repair events.  Join us, and contribute as little or as much time and brain-power you can.  Most of the tasks here are OK for non-technical people.  And we\'re always here to help.
+',
+        'number_of_tasks' => 'Number of tasks',
+        'number_of_quests' => 'Number of quests',
+        'all_volunteer_contributions' => 'Upstream volunteer contributions',
+        'all_quests' => '3 quests',
+        'my_contributions' => 'My contributions',
+        'my_quests' => ':value quests',
+        'join' => 'Join',
+    ],
+    'cta' => [
+        'title' => 'Help Now!  MobiFix',
+        'description' => '                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
```

Comment:
Intentional placeholder text?

Reply:
It is for now, waiting on the content from James.